### PR TITLE
fix #75238 gocmod.com

### LINKED ISSUE #75238: gocmod.com
### Issue URL (Anti Adblock Script)
 [https://www.gocmod.com/](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fwww.gocmod.com%2F)
### Comment
Username: `@XBadFellaX`
> 
> --
> The information below is automatically generated. Do not delete it.
> 
> Referrer:
> https://www.google.com/
> 
> Custom userscripts:
> https://userscripts.adtidy.org/release/popup-blocker/2.5/popupblocker.user.js
> https://userscripts.adtidy.org/release/assistant/4.3/assistant.user.js
> https://userscripts.adtidy.org/release/adguard-extra/1.0/adguard-extra.user.js
> https://userscripts.adtidy.org/release/adguard-wot/1.0/wot.user.js
> 

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | value
--- | ---
AdGuard product: | AdGuard for Windows v7.5.3
Browser: | Chrome
AdGuard driver: | TDI
DNS filtering: | server: `https://dns.adguard.com/dns-query`<br>
Filters: | <b>Ad Blocking:</b><br/> AdGuard Base<br/><br/><b>Privacy:</b><br/> AdGuard Tracking Protection<br/><br/><b>Social Widgets:</b><br/> AdGuard Social Media<br/><br/><b>Annoyances:</b><br/> Adblock Warning Removal List, <br/>AdGuard Annoyances<br/><br/><b>Other:</b><br/> AdGuard DNS, <br/>AdGuard Experimental<br/><br/><b>Language-specific:</b><br/> AdGuard Russian, <br/>AdGuard Dutch
Custom filter subscriptions: | https://block.energized.pro/blu/formats/filter

### PR DESCRIPTION
**Pull request by wout-codes**
#75238

### CODE REVIEW

File: EnglishFilter/sections/antiadblock.txt [hunk 1]
```diff
@@ -495,9 +495,10 @@ elpais.com,definitions.net,ensonhaber.com,lapatilla.com,sozcu.com.tr#%#//scriptl
 ||cdn.staticaly.com/gh/Arlina-Design/*/arlinablock.js
 ||rawcdn.githack.com/*/arlinablock.js
 ||cdn.jsdelivr.net/gh/dewaplokis/block@master/dewablock.js
+||cdn.jsdelivr.net/gh/*/js/arlinablock.js
 iptvbin.com$$script[tag-content="a.id=""arlinablock"""][max-length="1200"]
-||pagead2.googlesyndication.com/pagead/js/adsbygoogle.js$script,redirect=googlesyndication-adsbygoogle,important,domain=drivedrama.com|puressh.net|iptvbin.com
-@@||pagead2.googlesyndication.com/pagead/js/adsbygoogle.js$domain=drivedrama.com|puressh.net|iptvbin.com
+||pagead2.googlesyndication.com/pagead/js/adsbygoogle.js$script,redirect=googlesyndication-adsbygoogle,important,domain=gocmod.com|drivedrama.com|puressh.net|iptvbin.com
+@@||pagead2.googlesyndication.com/pagead/js/adsbygoogle.js$domain=gocmod.com|drivedrama.com|puressh.net|iptvbin.com
```
Comment (zloyden) [inline EnglishFilter/sections/antiadblock.txt:501]: Why did you add the domain to those rules?

Comment (wout-codes) [reply EnglishFilter/sections/antiadblock.txt:501]: @zloyden It's needed for the anti-adblock to be wiped out.

Comment (wout-codes) [reply EnglishFilter/sections/antiadblock.txt:501]: Okay it's needed. Why did I even add it there? I don't have any idea on why did I add it there.

**Review by zloyden — APPROVED** [review]
